cpp/src/io/csv/csv_gpu.cu

```diff
@@ -516,11 +525,13 @@ struct decode_op {
   /**
    * @brief Dispatch for all other types.
    */
-  template <typename T,
-            typename std::enable_if_t<!std::is_integral<T>::value and
-                                      !std::is_floating_point<T>::value> * = nullptr>
+  template <
+    typename T,
+    typename std::enable_if_t<!std::is_integral<T>::value and !std::is_floating_point<T>::value and
```

Comment:
We're using C++17 now so we can replace `std::is_integral<T>::value` with `std::is_integral_v<T>` in code we touch.

Reply:
Thanks for the hint! I've adopted it - also in a few other places 👍